server/tsconfig.json

```diff
@@ -5,11 +5,11 @@
         "module": "commonjs",
 
         /* Strict Type-Checking Options */
-        "strict": true,
+        "strict": false,
 
         /* Additional Checks */
-        "noUnusedLocals": true,
-        "noUnusedParameters": true,
+        "noUnusedLocals": false,
+        "noUnusedParameters": false,
```

Comment:
did this in both src/tsconfig and server/tsconfig

Reply:
dope sounds good!